doc/github_handler.rst

```diff
@@ -58,23 +58,36 @@ On this picture numbers depict the following events:
 
 1. User makes a new commit
 2. GitHub detects a commit, creates a check suite and sends a web-hook payload to Jenkins server
-3. A preconfigured Jenkins job passes received parameters to GitHub Handler
-4. GitHub Handler checks that repo name and web-hook event are applicable and and sends a request
+3. `Generic Webhook Trigger` plugin triggers a preconfigured Jenkins job ('GitHub Webhook handler') on an already set up
+   automation server, passing required parameters, such as payload contents and "x-github-event" header
+4. GitHub Handler checks that repo name and web-hook event are applicable, and if so, sends a request
    to create a new check run to GitHub
 5. GitHub creates a new check run and sends a new web-hook payload about this event
-6. GitHub Handler triggers a usual `Universum` run on an already set up automation server,
-   retrieving all required parameters from GitHub web-hook payload and passing them to the build
-7. Universum in default mode with ``--report-to-review`` on and ``--vcs-type=github`` uses received parameters
-   to report build ('check') start
-8. Depending on build result, Universum either reports build success or failure
-
-After this, check result can be viewed directly on GitHub.
+6. `Generic Webhook Trigger` plugin triggers the 'GitHub Webhook handler' job again, now with different event and
+   payload
+7. Based on received event, GitHub Handler triggers another preconfigured job with usual `Universum` call
```

Comment:
In my opinion it is better to keep details regarding 'Check commit' and universum in item 8.
Please also mention that the trigger is done using URL (http request).